js/opening-hours.functions.js

```diff
@@ -353,59 +718,76 @@ OpeningHours.prototype.calendarEvents = function (element, settings) {
 /**
  * Handle keyboard input to move to other dates.
  *
- * @param {Event} e
+ * @param {KeyboardEvent} event
  *   The keydown event.
- * @param {HTMLElement} elem
+ * @param {HTMLElement} element
  *   Wrapper element which contains the days.
  */
-OpeningHours.prototype.handleKeyboardInput = function (e, elem) {
-  let keyCode = e.keyCode || e.which;
-  let current = e.target;
-  let currentPosition = +current.getAttribute('aria-posinset');
+OpeningHours.prototype.handleKeyboardInput = function (event, element) {
+  var key = event.key;
+  var current = event.target;
+  var currentPosition = +current.getAttribute('aria-posinset');
 
-  const changeFocus = function () {
-    let nextElem;
-    let i = 0;
+  console.log(key);
```

Comment:
Let's not pollute every user's console 👍

Reply:
oops

Reply:
Removed